public/index.html

```diff
@@ -19,9 +19,9 @@
     <meta
       http-equiv="Content-Security-Policy"
       <% if (process.env.REACT_APP_CSP_ALLOW_UNSAFE_EVAL) { %>
-        content="script-src 'self' https://www.google-analytics.com https://www.googletagmanager.com 'unsafe-inline' 'unsafe-eval'"
+        content="script-src 'self' 'unsafe-inline' 'unsafe-eval'"
```

Comment:
Where in the code do we need to have `unsafe-eval`?

Reply:
I actually do not know yet, will need to look into it separately 🤔

Reply:
We do not, that is why this is under a conditional. `unsafe-eval` is used by istanbul to provide coverage reporting, so e2e test builds are built with `REACT_APP_CSP_ALLOW_UNSAFE_EVAL=true` so that istanbul can function.